src/FsAutoComplete.Logging/paket.references

```diff
@@ -2,3 +2,4 @@ FSharp.Core
 
 File: FsLibLog.fs
 
+Microsoft.NETFramework.ReferenceAssemblies
```

Comment:
Another way would be to add it via `directory.props`, so you can not forget to add it everywhere (And I guess the framework references will only change if new frameworks are added). But yeah.

Reply:
Mixing packagerefs from directory.props and paket feels a bit gross to me somehow

Reply:
:)